CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Change placement of repeatable decorator in the Inventory field mapping screens (UIDATIMP-518)
 * Implement delete row functionality for the Field Mapping Profile details (UIDATIMP-498)
 * Implement re-order row functionality for the Field Mapping Profile details (UIDATIMP-496)
+* Update logic for reference dropdowns (UIDATIMP-516)
```

Comment:
Seems like it's a bug, not a story. So, could you please move description to bugfix section

Reply:
done